### DISCUSSION TITLE
希望大家规范评论发言，回避政治话题！不要给受害者们带来不必要的麻烦

### DISCUSSION THREAD

**Discussion by bokomo** [WeNeedHome/SummaryOfLoanSuspension#537] [Ideas]
如题，请大家规范评论发言，回避政治话题！不要给受害者们带来不必要的麻烦！
政治话题只会带来更多的麻烦，而解决不了任何问题，希望大家的讨论仅围绕烂尾楼展开，拒绝成为极端神神和极端兔兔的发泄的窗口，不要给心怀恶意的人任何的可乘之机

**Comment by zxcs001** [comment]
你错了，是可以解决问题的。

当年江西这么一闹，农业税就取消了。

河南这么一闹，五万块钱一下的储户就都垫付了。

你不闹，你就一点统战价值都没有。就是要搞事，而且动静越大你才越有价值。同时你要尽可能的合法，保全自身安全。

这次停贷就是对共产党的一记重拳，我停贷合理合法，而且能让所有被烂尾楼祸害的人产生共鸣。

你一天不给我解决，我一天就不还钱。

---

有人回就多说几句。

而且你要理解什么是政治，政治就是个人争取自己利益的途径，是对自我权益诉求的表达。

不要自己把自己的思维给阉割了。现在就是想办法，怎么样才能把自己的房子要回来，怎么样才能让争取回来自己的权益。

但是目前，国内的情况就是这样。必须要戴着镣铐跳舞，现在唯一能找到的路径就是停贷，逼迫银行和政府给房地产公司施压，让他们完工。否则，压根没人会搭理你的。

你知道海南的烂尾楼么，烂了快三十年了，有解决么？你一辈子搭进去，就买套烂尾楼你甘心么？一辈子啊……

**Reply by z2046q** [reply]
有道理！

**Reply by wedxz** [reply]
要不说按闹分配呢

**Reply by liuwenwu520** [reply]
你说的还是有道理的，不过有一些是那种五毛党，蛊惑人心的家伙

**Reply by allenyan513** [reply]
讲得好

**Reply by z-huabao** [reply]
牛头不对马嘴！楼主说的是不要讨论共产党好坏的政治话题，你在这说要按闹分配、联合停贷，是，你说得对，但是和主题无关

**Reply by hrsgn** [reply]
> 牛头不对马嘴！楼主说的是不要讨论共产党好坏的政治话题，你在这说要按闹分配、联合停贷，是，你说得对，但是和主题无关

有问题都不能批评？那还怎么闹怎么分配

**Reply by lance111l** [reply]
烂尾楼是资金链断了   开发商没办法搞钱了   商业破产了     当时合同有担保就没问题

**Comment by b4o** [comment]
有些~祸就在带节奏，一看就是刚注册的号，到处仇视中国人，讨论问题就讨论问题，看看那些个像是被诅咒了的发言

**Reply by gitaa1** [reply]
看不惯可以不看，不认同可以讨论，既不想看也不想讨论可以离开，跑这里来堵人嘴真的没必要吧，在墙里堵的还不够吗？真不知道有些人是跑这里来干吗的，内鬼吗？

**Reply by b4o** [reply]
发了一大段文字 ，没一个词说烂尾这件事情，全是反着反那的

**Reply by gitaa1** [reply]
是不是到现在还不知道自己是怎么输的？到底是怎么被烂尾的就一点也不琢磨琢磨？

**Reply by zyh-future** [reply]
没看到仇视中国人的，你自己杜撰的？

**Comment by AlphaXu917** [comment]
特色资本主义

**Reply by duhanmin** [reply]
有资本主义特色的社会主义

**Reply by Siryedu** [reply]
> 有资本主义特色的社会主义

国家资本主义

**Reply by macqueen09** [reply]
封建资本主义

**Comment by user-hook-name** [comment]
你这问题的本身就是个政治问题.根本无法回避.
为什么烂尾?监管部门在哪?资金现在在哪?
追究到最后,是想逃避根源吗?

**Reply by wedxz** [reply]
谁会想到李佳琪会被莫名其妙的封杀,他也想岁月静好啊,他也不关心政治啊

**Reply by z-huabao** [reply]
抓住小偷就要打死吗？考虑后果了吗？同理，业主只想交房，不想坐牢啊！

**Comment by a6749920** [comment]
坦克开起来你以为是不存在的吗

**Comment by woshizengling** [comment]
没办法的，这些人就是来拱火的

**Comment by yalo-ddaa** [comment]
得了吧，当官的和资本家只关心你有没有影响他们的利益，谁关心你说什么啊。
河南储户们高举毛泽东的头像，然而对方关心了吗？

**Comment by program-thinked** [comment]
麻烦的怕是共产党了哈哈哈哈哈

**Comment by xiaer93** [comment]
企业挪用资金，银行违规放贷，监管不力。

这么久了期房还管理不好，尚不讨论各种豆腐渣工程。

是否要追责相关人员责任，是否要健全完善法律法规，是否希望今后合理合法的维权可以在国内公开讨论。

上述诉求属于政治范围吗？

**Reply by lance111l** [reply]
要追责相关人员责任，要健全完善法律法规    都是管理问题   不是政治 主义问题

**Comment by tencentbook** [comment]
为什么要回避政治话题？上学时就学过政治这门课，反而长大了不能关心国家大事了吗

**Comment by yalo-ddaa** [comment]
得了吧，不要自欺欺人了。
在老爷的眼里，你们侵犯到他们的利益，那你就是涉政。
就算你对着白衣人喊破喉咙”我爱国爱党“，他们也照样打你。

**Reply by leslieyuan** [reply]
真是跪久了站不起来了
一切问题都可以是政治问题, 回避啥呢?
你不去争取就不会给你解决, 烂尾这么多年国家不管, 你要联合起来停贷, 国家急了, 想清楚为啥了么?

**Comment by mehhss** [comment]
哪来的受害者，都是活该的，根本不值得同情

**Reply by pyman223** [reply]
同情心呢？

**Reply by MineCreeper86** [reply]
？

**Reply by GuoyiZhang** [reply]
他妈的，作为人矿，买了房子，烂尾了，还要被你说活该，你踏马是傻逼吧

**Reply by maxwillma** [reply]
就是，我也是 14亿分之一 ， 我怎么就买了美国的大豪斯，还不是你不够努力？

**Comment by clunyes** [comment]
这东西确实太敏感了

**Comment by sunyu559** [comment]
不去团结一切可以团结的力量。不想着追责，不想着维权。以为嘴上喊着“爱国爱党”房子就回来了。明明是一个政治问题，这个群体全部自我封闭，在国外的论坛上大喊杀死境外敌对势力，你们真是活该。这辈子都拿不回本属于自己的东西，哦不对是永世轮回都拿不回。

**Comment by volans19** [comment]
好家伙，要不是在github里我还以为你是国务院发言人了，你就是赵偷乐？
你还差一句没说，买了烂尾楼，你就偷着乐吧

**Comment by lance111l** [comment]
啥都能讲   有方法就行   光发牢骚没有用

**Comment by geralt2846** [comment]
这里可以随意骂脏话吗

**Comment by Cegazhos** [comment]
俺觉得无法回避。这种事情发生的根源就是中修给他们开发商开绿灯又弱监管弱法治造成的。扯到中修的责任了，那就是政治问题了。

**Reply by maxwillma** [reply]
本区被封了吗？

**Reply by Cegazhos** [reply]
> 本区被封了吗？

中修半年前急轉彎了。台灣鄉巴佬上網不看新聞?

**Reply by Tentiauhong** [reply]
大家懷疑這位是神經病請按讚

**Comment by Keyuandxie** [comment]
呵呵呵 换一个党也没有共产党这么好的 共产党能脱贫你们能吗

**Reply by ghost** [reply]
共产党当然可以脱贫了，哪个共产党员穷的？哪个书记不是身价千亿？

**Reply by maxwillma** [reply]
你这回答，无懈可击

**Reply by b4o** [reply]
> 共产党当然可以脱贫了，哪个共产党员穷的？哪个书记不是身价千亿？

你说哪个政党哪个国家能让全国家家户户能路通电通网？能让全国80%的人满意？你家的菜一一，40%就能当选总统太可笑了

**Reply by XiaoNianPoPo** [reply]
80%的中国人没有达到小康水平你知道么

**Reply by maxwillma** [reply]
人家 40%  老农民 退休金都 5000 软妹币了，80% 的话 ，想都不敢想。我看还是金太阳的100% 支持率好。LOL

**Comment by b4o** [comment]
不要天真了，除了大陆，所有地方只要有中国人的地方都充满了台独港独

**Reply by maxwillma** [reply]
不认同CCP的统治就是X独吗？

**Comment by a6749920** [comment]
80%满意估计是全国的公务员事业单位的人，农民不算



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年6月9日(星期五) 下午5:17
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [WeNeedHome/SummaryOfLoanSuspension] 希望大家规范评论发言，回避政治话题！不要给受害者们带来不必要的麻烦 (Discussion #537)





  
共产党当然可以脱贫了，哪个共产党员穷的？哪个书记不是身价千亿？
  
你说哪个政党哪个国家能让全国家家户户能路通电通网？能让全国80%的人满意？你家的菜一一，40%就能当选总统太可笑了

**Comment by lance111l** [comment]
满意是希望自己的付出比得到的少  这靠自己不行   需要团队和社会组织提高效率才行   不付出当然没有回报   农民出来打工是好事   出国打工也很多   只是别忘本了   没有祖宗的庇护  没有文化 没有国家的支持  你没有未来
